app/src/main/res/values/strings.xml

```diff
@@ -81,7 +82,7 @@
     <string name="code_signature_found">We have found code signature of the following trackers in the application:</string>
     <string name="code_signature_not_found">We have not found code signature of any tracker we know in the application.</string>
     <string name="tracker_info">A tracker is a piece of software meant to collect data about you or your usages.</string>
-    <string name="analysed">This app hasn\'t been analysed by Exodus Privacy yet.</string>
+    <string name="analyzed">This app hasn\'t been yet analyzed.</string>
```

Comment:
Add new string because this string has already use on app detail fragment

Reply:
Done 6b6b0b97c6a27696e7335800b1106e1aed77b138